### DISCUSSION TITLE
Show rosbag size and/or bandwidth by each topic

### DISCUSSION THREAD

**Discussion by Combinacijus** [foxglove/community#428] [Q&A]
It's would be nice to see rosbag sizes for each topic this would help to find which topics uses most storage and optimize them. Also In 2nd image it could be sorted by pressing label

Example images where it could be put:
![image](https://user-images.githubusercontent.com/16375702/198375006-9d64c173-46b8-4015-90f5-c8e674385563.png)
![image](https://user-images.githubusercontent.com/16375702/198376417-d75077b3-ae0b-45e0-8953-e8a475537d64.png)

Implementation in ros rqt `rosrun rqt_topic rqt_topic`:
![image](https://user-images.githubusercontent.com/16375702/198377005-d8fbcf14-1593-4b9d-b0ea-00e2e4e6f649.png)

**Comment by ndmmdn** [comment]
Any update on this? I would really appreciate such a functionality, ideally also when livestreaming from a robot
